src/t5/t5_model.rs

```diff
@@ -133,6 +133,8 @@ pub struct T5Config {
     pub feed_forward_proj: Option<FeedForwardProj>,
     pub tie_word_embeddings: Option<bool>,
     task_specific_params: Option<TaskSpecificParams>,
+    pub output_attentions: Option<bool>,
```

Comment:
Regardless of the decision on the attentions inclusion for the sentence emebeddings, please keep these changes, and thank you again for catching this.